README.md

```diff
@@ -121,64 +122,72 @@ You can register/login in the app using our internal protocol, or OAuth2 and goo
     }
     ```
 
+    Excepts:
+    - 401: Auth token not valid
+    - 400: filter not valid
+
 - `/api/get_site_info` [GET]
 
   the requests expect the parameter as an argument:
-    - `filter` -> all, visited, unvisited
+    - `id` -> id of the place
+
+  if id is valid, returns:
+  ```
+  {
+    "city": str,
+    "description": str,
+    "employees": 
+      [ # Only if employees are assigned 
+        {
+        "added_by": int,
+        "can_reward": bool,
+        "email": str,
+        "first_name": str,
+        "last_name": str,
+        "place_id": int,
+        "profile_picture": str 
+        },
```

Comment:
JSON doesn't allow comma as last character and this is more explicit that it's a list.
```suggestion
        }, ...
```